deploy/test/test_cases/TEST_ID_18_helm_multiple_provider_multiple_secrets.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This test verifies that two Secrets Provider Jobs deploy successfully in the same namespace
+# TODO verify Job start up and completed logs
+
+setup_helm_environment
+
+echo "Create second secret"
+create_k8s_secret_for_helm_deployment "another"
+set_secret secrets/another_test_secret another-some-secret-value
+
+pushd ../../../
+  fill_helm_chart
+  helm install -f "secrets-provider/ci/test-values.yaml" \
+    secrets-provider ./secrets-provider \
+    --set-file environment.conjur.sslCertificate.value="deploy/test/test_cases/conjur.pem"
+
+  export SECRETS_PROVIDER_ROLE=another-secrets-provider-role
+  export SECRETS_PROVIDER_ROLE_BINDING=another-secrets-provider-role-binding
+  export SERVICE_ACCOUNT=another-secrets-provider-service-account
+  export K8S_SECRETS=another-test-k8s-secret
+  export SECRETS_PROVIDER_SSL_CONFIG_MAP=another-secrets-provider-ssl-config-map
+  fill_helm_chart "another-"
+  helm install -f "secrets-provider/ci/another-test-values.yaml" \
+    another-secrets-provider ./secrets-provider \
+    --set-file environment.conjur.sslCertificate.value="deploy/test/test_cases/conjur.pem"
```

Comment:
what exactly are we testing here? can we use the same role and rolebinding to get the same effect? can we use the same service-account? can we use the same config map?

Reply:
unfortunately we cannot have 2 of the same k8s resource. We will get "already exists" error from k8s so if we want another secrets provider we need to create a new set of rbac names

Reply:
so you are saying that 2 secrets-providers can't use the same service account?

Reply:
logically it will work to have the same service accoun but @eladkug and I wanted to test the scenario where we have two separate groups of resources

Reply:
what's so special about that scenario? why do we need to test it? don't we prefer to test the scenario where both secrets-providers have the same service-account? it is closer to a customer env

Reply:
@eladkug what do you think?

Reply:
I think we should have tests both for common service account and for 2 separate ones

Reply:
i agree with @oburstein-hub . Different scenarios.. Thanks @orenbm  for raising this.

Reply:
I want to preface that this is not a natural flow because each secrets provider is deployed via Helm which triggers the creation of completely new resources each time. We can _however_ make a test to deploy a single secrets provider and for the next secrets provider job, set `CREATE_SERVICE_ACCOUNT=false` and the customer gives us the name of the previous ServiceAccount created but I am not sure what is the most "normal" flow. 

I will create a test for the latter

Reply:
Is it enough just RBAC (service account, rolebinding, role, etc) to be the same? or ALL resources? (config map, etc)

Reply:
just RBAC